src/models/ProfileToFormRPartBInitialValues.ts

```diff
@@ -22,9 +22,12 @@ export function ProfileToFormRPartBInitialValues(
     programme && programme.curricula.length > 0
       ? programme.curricula
           .filter(c => c.curriculumSubType === MEDICAL_CURRICULUM)
-          .reduce(function (a, b) {
-            return a.curriculumStartDate > b.curriculumStartDate ? a : b;
-          })
+          .sort(
+            (a, b) =>
+              new Date(b.curriculumStartDate).getTime() -
+              new Date(a.curriculumStartDate).getTime()
```

Comment:
Unsure why I needed to initialize a new date here, but `curriculum.curriculumStartDate.getTime()` errored.

Reply:
more magic